app/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.test.tsx

```diff
@@ -270,3 +242,44 @@ describe("FormAudience", () => {
     expect(screen.queryByTestId("missing-duration")).toBeInTheDocument();
   });
 });
+
+const renderSubjectWithDefaultValues = (onSubmit = () => {}) => {
+  render(
+    <Subject
```

Comment:
Hmm, if these are default values, then they should probably be defaults within the `Subject` component.

But, they weren't before this PR, so I wouldn't make it a snag for merging

Reply:
Hmm yeah, they're defaults with a fresh experiment. That would make the most sense, though the Subject mock is pulling from the values we have in `mockExperimentQuery`. I think we'd need some test refactoring :/ I think we've just stuck whatever made the most sense for tests at the time into `mockExperimentQuery`. I'm going to leave this as-is, we _could_ make a ticket for updating what `mockExperimentQuery` to reflect what's default, but I have a feeling we'd never get to it because it'd require a lot of test alterations